ethstore/src/random.rs

```diff
@@ -37,3 +38,25 @@ impl Random for [u8; 32] {
 		result
 	}
 }
+
+/// Generate a string which is a random phrase of a number of lowercase words.
+///
+/// `words` is the number of words, chosen from a dictionary of 7,530. An value of
+/// 12 gives 155 bits of entropy (almost saturating address space); 20 gives 258 bits
+/// which is enough to saturate 32-byte key space
+pub fn random_phrase(words: usize) -> String {
+	lazy_static! {
+		static ref WORDS: Vec<String> = String::from_utf8_lossy(include_bytes!("../res/wordlist.txt"))
```

Comment:
do you think this could be compressed somehow or hidden behind a feature gate? this is a substantial addition to the binary size which our embedded users might not appreciate much